packages/components/src/highlight-cards/style.scss

```diff
@@ -30,7 +30,7 @@ $vertical-margin: 32px;
 
 	.highlight-card {
 		border-color: var(--studio-gray-5);
-		border-radius: 4px;
+		border-radius: 5px; // stylelint-disable-line scales/radii
```

Comment:
I think leaving the `border-radius: 4px` looks good to me, which corresponds to the `stylelint` rules. 😄

Reply:
I'll leave this alone for now 🙂